src/components/navigation-header/components/product-page-content/utils/__tests__/get-nav-items.test.ts

```diff
@@ -0,0 +1,161 @@
+import waypointProductData from 'data/waypoint.json'
+import terraformProductData from 'data/terraform.json'
+import { getNavItems } from '../get-nav-items'
+import { ProductData } from 'types/products'
+
+describe('getNavItems', () => {
```

Comment:
I've added tests for `getNavItems` with inline snapshots. Happy to adjust the testing approach if we have something else in mind - for me personally, inline snapshots felt like a good balance given the return data.